src/draw/nxp/pxp/lv_pxp_osa.c

```diff
@@ -81,27 +98,12 @@ static pxp_cfg_t _pxp_default_cfg = {
  *   GLOBAL FUNCTIONS
  **********************/
 
-void PXP_IRQHandler(void)
+#if defined(__ZEPHYR__)
```

Comment:
Please remove this line, it will break the freertos support.

Reply:
Within LVGL this is not used - is this something that needs to be callable from the outside?

Reply:
`PXP_IRQHandler()` is what the name suggest - the handler of the PXP IRQ.

It is similar to what you already added `PXP_ZephyrIRQHandler` except that in case of FreeRTOS the PXP handler is mapped in the vector table outside of the lvgl code.

Reply:
Okay, so that function _is_ called from outside? I wasn't clear on that.

Even though it wasn't perfect, I approved yesterday because it was harmless and close enough. But the naming is still confusing because `PXP_IRQHandler` is actually called from outside and the name is important, because it maps to something in the symbol table as you say, while `PXP_ZephyrIRQHandler` is not special yet has a name that makes it seem like it maps to something in the symbol table.

Reply:
In this case we can rename the `PXP_ZephyrIRQHandler` as this is a static function.
But `PXP_IRQHandler` need to stay in place with exactly the same name.

Thanks.

Reply:
Reverted this part of the change moved the zephyr handler to the static section.